src/components/atoms/InfoBox/infoBox.atom.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import {EColor, EFontWeight} from "../../../shared/types/styles.types";
+import Skeleton from "../Skeleton/skeleton.atom";
+import Typography from "../Typography/typography.atom";
+import { IInfoBox } from "./infoBox.atom.types";
+import styles from "./infoBox.atom.module.scss";
+
+const InfoBox: React.FC<IInfoBox> = (props) => {
+  const {usdValue, value, footer, loading = false} = props;
+  // temp - find a better way to handle this
+  const showUsdValue = usdValue && usdValue !== "$0"; 
```

Comment:
I think this is on parent component to decide if it wants to pass something to the optional prop or not. If something is passed we just render it here, otherwise no render

Reply:
makes sense. Will add a showHeader flag on the interface